src/lib/get-marked-with-highlighter.ts

```diff
@@ -1,20 +1,18 @@
 import { getLanguage, highlight } from 'highlight.js';
 import marked, { MarkedOptions } from 'marked';
 
-export const getMarked = (options: MarkedOptions) => {
+export const getHighlightRenderer = (options: MarkedOptions) => {
 	const renderer = options.renderer ?? new marked.Renderer();
 
 	// only add if the renderer has no custom `code` property yet
 	if (!Object.prototype.hasOwnProperty.call(renderer, 'code')) {
-		renderer.code = (code, language) => {
+		renderer.code = (code: string, language: string | undefined) => {
```

Comment:
Those types are provided and don't need to be set explicitly.

```suggestion
		renderer.code = (code, language) => {
```